pagopa/io-app
# chore: [IOCOM-1580] Fims auth missing service data error case

## Short description
addition of said error case, along a refactor 

<img width="250" alt="Screenshot 2024-09-04 at 15 21 04" src="https://github.com/user-attachments/assets/a9f76345-01b7-4b98-af5a-606143c90243">


## List of changes proposed in this pull request
- required I18n
- refactor to split the code in different files, in order to improve readability
- addition of said error case
- small refactor in the autoFetch service hook

## How to test
using the dev-server and proxyman, navigate to the FIMS SSO message details, force the next call to `services/service1` to return 404 or any error status, and make sure the picured screen is shown.
also make sure the normal flow works as intended.
